### DISCUSSION TITLE
Weather forecast expired?

### DISCUSSION THREAD

**Discussion by wimplo79** [weather-gov/api#715] [General]
Hello, I pull data from the API for a localhost command center for my family via a Raspberry Pi, etc.  Anyways, everything had worked great for quite awhile.  Now I just got a 503 error on the console and it says from forecast grid has expired.  I am just wondering if this is just a temporary hiccup on the API side of things or if there is something I need to do on my end to refresh the forecast data every so often outside of refreshing the website to pull the new data?  Meaning adding something to my javascript with the header that includes my email for contact?  Here is the message I got.  

{
    "correlationId": "f1a4adb",
    "title": "Forecast Grid Expired",
    "type": "https://api.weather.gov/problems/ForecastGridExpired",
    "status": 503,
    "detail": "The requested forecast grid was issued 2024-04-17T08:29:53+00:00 and has expired.",
    "instance": "https://api.weather.gov/requests/f1a4adb"
}

Here is my API forecast URL.

https://api.weather.gov/gridpoints/MPX/97,65/forecast

Thanks in advance for any assistance and thanks for providing weather data!

**Comment by dwhitemv25** [comment]
The API folks can chime in here, but I think this is a problem with the data provided by the WFO -- they didn't upload the next forecast package soon enough and the old one expired. There's nothing you can do short of calling your WFO and letting them know. 

There have been lots of issues with NWS systems due to a cooling failure in the Boulder datacenter this week and it may be related to that.

Going back through my error log, I've seen this event twice from my WFO since August 8, 2022, so this problem should be very rare.
